README.md

```diff
@@ -420,31 +279,215 @@ The temperature data is stored in files under data folder. One file is made for
+
+    ```bash
+    sh flash_border_router.sh <board-id>
+    ```
+
+    ```bash
+    # Example to get correct <board.id>
+
+    user@grenoble.iot-lab.info:~$ iotlab-experiment --jmespath="items[*].network_address | sort(@)" get --nodes
+    [
+    "m3-6.grenoble.iot-lab.info",
+    "m3-7.grenoble.iot-lab.info"
+    ]
+    # In this example <board-id> = 6 or 7
+    # Only need to pick one ID!
+    ```
+
+4. Check which tap interfaces are open:
+
+    ```bash
+    ip addr show | grep tap
+    ```
+
+5. Check which ipv6 prefixes are already used:
+
+    ```bash
+    ip -6 route
+    ```
+
+6. Launch the ethos_uhcpd command with:
+
+    ```bash
+    sudo ethos_uhcpd.py m3-<id> tap<num> <ipv6_prefix>::/64
+    ```
+
+    ```bash
+    # Example to get correct <id>, <num> and <ipv6_prefix>
+    
+    # For <id>:
+    # Use the same ID as you chose earlier with command: sh flash_border_router.sh <board-id>
+
+    #For <num>:
+    user@grenoble.iot-lab.info:~$ ip addr show | grep tap
+    313: tap0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc pfifo_fast state DOWN group default qlen 1000
+    ...
+    316: tap1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc pfifo_fast state DOWN group default qlen 1000
+    ...
+    582: tap8: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc pfifo_fast state DOWN group default qlen 1000
+    ...
+    # Example list of network interfaces already in use. This example has 3 network interfaces (eg. tap0, tap1 and tap8). 
+    # Choose a free interface, for example <id> = tap8 + 1 = 9
+
+    #For <ipv6_prefix>
+    user@grenoble.iot-lab.info:~$ ip -6 route
+    2001:660:5307:3100::/64 via fe80::2 dev tap0 metric 1024 linkdown  pref medium
+    2001:660:5307:3102::/64 via fe80::2 dev tap2 metric 1024 linkdown  pref medium
+    2001:660:5307:3103::/64 via fe80::2 dev tap3 metric 1024 linkdown  pref medium
+    2001:660:5307:3104::/64 via fe80::2 dev tap4 metric 1024 linkdown  pref medium
+    2001:660:5307:3107::/64 via fe80::2 dev tap100 metric 1024 linkdown  pref medium
+    ...
+    # Example list of ipv6 prefixes in use. 
+    # Choose a free prefix, for example <ipv6_prefix> = 2001:660:5307:3108
+    ```
+
+7. After launching ethos_uhcpd, leave the script running!
+
+### Flash iot-lab node
+
+To flash the application to iot-lab node, follow these guidelines:
+
+1. Open new ssh connection to grenoble.iot-lab.info:
+
+    ```batch
+    ssh <username>@grenoble.iot-lab.info
+    ```
+
+2. Set source:
+
+    ```bash
+    source /opt/riot.source
+    ```
+
+3. Change into the miniproject-1 repository working directory:
 
     ```bash
     cd IoT-miniproject-1/app
     ```
 
-3. Flash the firmware into the experiment node:
+4. Flash the firmware into the experiment node. Use different `<board-id>` than for border router.
 
     ```bash
-    make IOTLAB_NODE=m3-114 DEFAULT_CHANNEL=18 DEFAULT_PAN_ID=0xbcb4 flash 
+    make IOTLAB_NODE=m3-<board-id> DEFAULT_CHANNEL=18 DEFAULT_PAN_ID=0xbcb4 flash 
     ```
```

Comment:
add nc command and new terminal for that

Reply:
Added